### PR TITLE
Added Custom-Range-Slider

### PR DESCRIPTION
**Pull request by Atif0604**
Hey @Ayushparikh-code @chicken-biryani 
This PR will Resolve the issue => Custom Range Slider #38
Please Review my PR

### CODE REVIEW

**Comment by chicken-biryani** [comment]
@Atif0604 could you please add a README file?

**Comment by chicken-biryani** [comment]
The ReadMe file should contain the following :

Contents of the README:

-Project overview.
-Use of the project.
-Stack of your project.
-How to set up / run your project.
-Screenshots of the pages/output.
(A short description under each screenshot... Describing the page a little bit)
This would help visitors/developers in general.

@Atif0604

**Comment by Atif0604** [comment]
> The ReadMe file should contain the following :
> 
> Contents of the README:
> 
> -Project overview.
> -Use of the project.
> -Stack of your project.
> -How to set up / run your project.
> -Screenshots of the pages/output.
> (A short description under each screenshot... Describing the page a little bit)
> This would help visitors/developers in general.
> 
> @Atif0604

Its Already There @chicken-biryani

File: Custom-Range-Slider/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git
+```
+- Go to the directory
+- Run the index.html file
+- Set the Range According to your wish
+
+
+<h3> ScreenShots </h3>  
+<img src = "Custom-Range-Slider.png" alt="Sample Image">
+<br>
```
Comment (chicken-biryani) [inline Custom-Range-Slider/README.md:26]: Apply markdown beautifier. Use link https://codebeautify.org/markdown-formatter

Also can you please put the image, the link isn't working, 
Use link
https://stackoverflow.com/questions/14494747/how-to-add-images-to-readme-md-on-github

File: Custom-Range-Slider/index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                <div class="value"></div>
+            </div>
+        </div>
+    </div>
+    <div  class="footer">
+        <h4>Designed & Developed with &#10084;&#65039; by Atif</h4>
+    </div>
+    <script src="./script.js"></script>
+</body>
+</html>
```
Comment (chicken-biryani) [inline Custom-Range-Slider/index.html:50]: Apply an Html beautifier to this code:
https://codebeautify.org/htmlviewer --- use this link.

**Comment by chicken-biryani** [comment]
@Atif0604 my bad.

File: Custom-Range-Slider/index.html (hunk 2 above)
Comment (Atif0604) [reply Custom-Range-Slider/index.html]: Done @chicken-biryani

File: Custom-Range-Slider/README.md (hunk 1 above)
Comment (Atif0604) [reply Custom-Range-Slider/README.md]: Done @chicken-biryani

**Review by Ayushparikh-code — APPROVED** [review]

File: Custom-Range-Slider/README.md (hunk 1 above)
Comment (chicken-biryani) [reply Custom-Range-Slider/README.md]: @Atif0604 The image is still not working. Can u please have a look into that. The rest looks good to go. Just make the image work.

Comment (Atif0604) [reply Custom-Range-Slider/README.md]: I made the some changes @chicken-biryani 
this is the bulid what i see now in my repo 
image

File: Custom-Range-Slider/index.html (hunk 2 above)
Comment (chicken-biryani) [reply Custom-Range-Slider/index.html]: Yep its good to go now.

**Comment by Atif0604** [comment]
Hey @Ayushparikh-code @chicken-biryani @harshita2216 
Please Add LGMSOC21 label to this PR that it should be showing in the label section of PR as my PR is not being counted just because the LGMSOC21 label is not showed up in label section

**Pull request merged**